### ISSUE TITLE
Understanding Softmax loss function backpropagation In Depth - 海龙的博客 | Mastering Artificial Intelligence

### ISSUE THREAD

**Issue by lotuswhl** [lotuswhl/lotuswhl.github.io#8]
https://lotuswhl.top/2018/06/03/understand-softmax-bp-in-depth/ 

 As the saying goes,Where there is a will,there is a way
